src/content/docs/bot-protection/reference.mdx

```diff
@@ -329,6 +331,32 @@ their "advertising quality" bot.
   <SvelteKitFiltering slot="sveltekit" />
 </SlotByFramework>
 
+## Bot verification
+
+We support verifying the authenticity of requests from identifiable bots by
```

Comment:
Can you rephrase this slightly so it's clear we're doing this behind the scenes for you and you don't need to do the lookups yourself?

Reply:
I've added an extra sentence to explain it, let me know what you think!

Reply:
Looks good, thanks.